packages/flutter/test/widgets/routes_test.dart

```diff
@@ -864,6 +864,133 @@ void main() {
       expect(nestedObserver.dialogCount, 1);
     });
   });
+
+  group('ModalRoute', () {
+    testWidgets('default barrierCurve', (WidgetTester tester) async {
+      await tester.pumpWidget(MaterialApp(
+        home: Material(
+          child: Builder(
+            builder: (BuildContext context) {
+              return Center(
+                child: RaisedButton(
+                  child: const Text('X'),
+                  onPressed: () {
+                    Navigator.of(context).push<void>(
+                      _TestDialogRoute<void>(
+                        child: const Text('Hello World'),
+                      )
+                    );
+                  },
+                ),
+              );
+            }
+          ),
+        ),
+      ));
+
+      final CurveTween _defaultBarrierTween = CurveTween(curve: Curves.ease);
+      double _getBarrierTweenAlphaValue(double t) {
```

Comment:
Maybe rename to "getExpectedBarrierAlphaValue"?

(I was confused because this didn't seem to access the actual value from the barrier).